TestingLowerBounds/FDiv/CondFDiv.lean

```diff
@@ -274,9 +281,33 @@ lemma condFDiv_nonneg [IsMarkovKernel κ] [IsMarkovKernel η]
   simp only [EReal.coe_nonneg]
   apply integral_nonneg _
   intro x
-  have h := @fDiv_nonneg _ _ (κ x) (η x) f _ _ hf_cvx hf_cont hf_one
+  have h := fDiv_nonneg (μ := κ x) (ν := η x) hf_cvx hf_cont hf_one
   simp [EReal.toReal_nonneg, h]
 
+lemma condFDiv_const {ξ : Measure β} [IsFiniteMeasure ξ] (h_ne_bot : fDiv f μ ν ≠ ⊥) :
```

Comment:
Call this one `condFDiv_const'` and add another one called `condFDiv_const` with an added hypothesis `IsFiniteMeasure μ`, but without the hypothesis `h_ne_bot` (which then follows from `fDiv_ne_bot`).